test/end-to-end-tests/start.ts

```diff
@@ -112,7 +112,7 @@ async function runTests() {
         /**
          * TODO: temporary only use one user session data
          */
-        performanceEntries = JSON.parse(measurements);
+        performanceEntries = JSON.parse(measurements ?? "[]");
```

Comment:
this fixes a non-fatal error when the tests fail very early on, like during the first registration.